test/spec/quiet/commented-djs/expect.js

```diff
@@ -1,3 +1,3 @@
 /*
- very x is y
+very x is y
```

Comment:
only two tests that were borked. I'm actually thinking that maybe `quiet`/`loud` should format things:

```
/*
 * very x is y
 */
```

Reply:
hmm, comments might be a case where we should preserve the spacing, if they're included at all

Reply:
makes sense.

I'll see what i can come up with to try to pass this around so we can recursively consume keys!